nbconvert/preprocessors/extractoutput.py

```diff
@@ -54,6 +54,11 @@ class ExtractOutputPreprocessor(Preprocessor):
         {'image/png', 'image/jpeg', 'image/svg+xml', 'application/pdf'}
     ).tag(config=True)
 
+    escape_dir_spaces = Bool(
+        False, help="If True, spaces will be replaced with underscores in the "
+        "output directory."
+    ).tag(config=True)
```

Comment:
This is a slippery slope, which bends into the larger PR which maps everything to a tmp directory with safe file naming before copying back. It's also not clear if this is a legal operation change as a default.

Reply:
I'd maybe skip this or separate it into another PR for temporarily mapping files to safe paths before copying back to the actual output_dir

Reply:
From my knowledge of the larger PR. It only fixes things for the pdf exporter. It someone converted to latex and then manually compiled with latex, then paths would create issues still. This is a partial fix, and IMHO is good to do for the pure latex.

Also, if a latex error is raised the pdf will not be saved. I don't know why, since latex should be saving the pdf. This prevents a regression of notebooks that used to convert (but with a visual glitch) don't anymore. I need to look further into why the pdf isn't saving.

Reply:
Ah, I do see that in the larger PR there is this ```true_output_files_dir = resources.get('output_files_dir', cwd)``` If I pass the new escaped ```output_files_dir``` back to the resources then this would work.